Patch by Hironori Kitagawa to fix wrong sizes of mpost
---
 texk/web2c/mplibdir/svgout.w |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- texlive-bin.orig/texk/web2c/mplibdir/svgout.w
+++ texlive-bin/texk/web2c/mplibdir/svgout.w
@@ -986,7 +986,7 @@
    }
   }  else {
     mp_svg_open_starttag(mp, "text");
-    ds=(mp->font_dsize[gr_font_n(p)]+8) / 16;
+    ds=(mp->font_dsize[gr_font_n(p)]+8) / 16 / 65536.0;
     mp_svg_store_double(mp,ds);
     mp_svg_attribute(mp, "font-size", mp->svg->buf);
     mp_svg_reset_buf(mp);
